Yafc.Model/Model/ProductionTable.cs

```diff
@@ -74,19 +74,21 @@ private void Setup(List<RecipeRow> allRecipes, List<ProductionLink> allLinks) {
             }
         }
 
-        private static void ClearDisabledRecipeContents(RecipeRow recipe) {
+        private static void ClearRecipeContents(RecipeRow recipe, bool recurse) {
```

Comment:
If i read this right, calling this method with (recipe, false) is short hand for 

```C#
            recipe.recipesPerSecond = 0;
            recipe.parameters = RecipeParameters.Empty;
```
?
if so, i would just extract a method for these 2 lines, and replace all calls with `ClearRecipeContents(recipe, faslse);` with that new method. Removing this boolean from this methors parameters. I dont think i saw any place in the code where this method is called with a variable instead of a constant.